### ISSUE TITLE
Rearchitect/Refactor for Seamless 32/64 Bit

### ISSUE THREAD

**Issue by zcanann** [Squalr/Squalr#134]
The current issues come from the following:
- Running Fasm compiler from 64 bit (this is a 32 bit library to which I lack the source code -- kinda)
- Microsoft .NET heap walking a 32 bit process from a 64 bit (currently not possible)

The current solution is to create a 64 bit and 32 bit executable that run in the background alongside Ana. Any commands that have specific bit requirements are sent to the corresponding service.

Basically I need to find the fasm source, figure out how to compile a microsoft coff com object, and create a 32 and 64 bit dll respectively. OR port to C# entirely (I may do this in the far far far far far future)

Or just switch to NASM or MASM if they can be used easier?

For the .Net heap walking, I will need to refactor Microsoft's shitty code

**Comment by zcanann** [comment]
Turns out the AsmJit library solves the assembler issues. And this greatly improves scripting for many, many reasons.

As for the .Net library, I'll have to fix their code for them probably.
